students/BRACH_Jakub/lesson_01_basics/L01P02_Triangles.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+print("Enter the base length")
+base = input()
+print("Enter the height")
+height  = input()
+
+if( height>0 and base > 0):
```

Comment:
Parentheses are unnecessary here. Spacing is wrong.
Could look like:
`if height > 0 and base > 0:`

Reply:
Done.